packages/compartment-mapper/src/types.js

```diff
@@ -190,6 +197,12 @@ export const moduleJSDocTypes = true;
  * }>}
  */
 
+/**
+ * @typedef {Object} ParserImplementation
+ * @property {boolean} heuristicImports
```

Comment:
Please add a description of `heuristicImports` and the motivation for its existence.